### PR TITLE
Xcode 14 - set CODE_SIGNING_ALLOWED to NO

### PR DESCRIPTION
**Pull request by steve-sun-instacart**
I was not able to compile the project via Xcode 14 due to 
```
Cannot code sign because the target does not have an Info.plist file and one is not being generated automatically. Apply an Info.plist file to the target using the INFOPLIST_FILE build setting or generate one automatically by setting the GENERATE_INFOPLIST_FILE build setting to YES (recommended).
```

Setting `CODE_SIGNING_ALLOWED` to `NO` to compile on Xcode 14 because Xcode 14 requires bundle needs to sign

### CODE REVIEW

File: FontAwesome.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -162,6 +166,7 @@
 				CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF = YES;
 				CLANG_WARN_OBJC_LITERAL_CONVERSION = YES;
 				CLANG_WARN_OBJC_ROOT_CLASS = YES_ERROR;
+				CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER = YES;
```
Comment (satish-ik) [inline FontAwesome.xcodeproj/project.pbxproj:169]: did these things get auto updated when you opened the project in xcode 14?

**Review by satish-ik — APPROVED** [review]
verified running `carthage build --archive --verbose` on this branch and build succeeded, good to 🚀

File: FontAwesome.xcodeproj/project.pbxproj (hunk 1 above)
Comment (steve-sun-instacart) [reply FontAwesome.xcodeproj/project.pbxproj:169]: Yeah. Which is weird. Let me try another thing

Comment (steve-sun-instacart) [reply FontAwesome.xcodeproj/project.pbxproj:169]: No, after setting `CODE_SIGNING_ALLOWED`

Comment (steve-sun-instacart) [reply FontAwesome.xcodeproj/project.pbxproj:169]: We didn't ignore `xcodeproj` in `.gitignore` that's why all these weird things are here :(

Comment (joeydong) [reply FontAwesome.xcodeproj/project.pbxproj:169]: this is the intended workflow. carthage cannot execute xcodegen before compiling so we:
1. Update XcodeGen's project.yml file
2. Execute XcodeGen to generate xcodeproj file
3. Check in all changes

**Pull request merged**